### PR TITLE
Adding readme with a quickstart 

### PR DESCRIPTION
**Pull request by mansona**
I just ran these commands and it worked amazingly 🎉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,28 @@
...
+and the rest of the work is performed by EmberJS' build system.
+
+## Quick Start
+
+```sh
+npm init ember-app super-blog
+
+cd super-blog
+
+npx ember install empress-blog empress-blog-casper-template
```
Comment (lenrsmith) [inline README.md:16]: Should this be empress-blog-w3css-blog-template?

**Review by lenrsmith — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mansona) [reply README.md:16]: yep sorry 🙈 copy-pasta error 🍝

Comment (lenrsmith) [reply README.md:16]: No worries. I tweaked it. Thanks for all the help.

Comment (mansona) [reply README.md:16]: no, thanks for doing all of this 🎉 you are the first person to create a custom theme for empress-blog and that is pretty exciting 😍

**Pull request merged**